plutus-conformance/src/Common.hs

```diff
@@ -1 +1,21 @@
-module PlutusConformance.Common where
+module Common where
+
+import PlutusCore.Core (defaultVersion)
+import PlutusCore.Default (DefaultFun, DefaultUni)
+import PlutusCore.Evaluation.Machine.ExBudgetingDefaults (defaultCekParameters)
+import PlutusCore.Evaluation.Result (EvaluationResult)
+import PlutusCore.Name (Name)
+import Prelude
+import UntypedPlutusCore.Core.Type qualified as UPLC
+import UntypedPlutusCore.Evaluation.Machine.Cek (unsafeEvaluateCekNoEmit)
+
+type UplcProg = UPLC.Program Name DefaultUni DefaultFun ()
+
+termToProg :: UPLC.Term Name DefaultUni DefaultFun () -> UplcProg
+termToProg = UPLC.Program () (defaultVersion ())
```

Comment:
I think we have many duplicates of such a helper function.  e.g. here:
https://github.com/input-output-hk/plutus/blob/master/plutus-ledger-api/test/Spec/Eval.hs#L164-L165

We should standardize it somewhere; perhaps in a uplc/core/type.hs? anyway, not very important

Reply:
yeah, good idea. I'll do that.